contracts/teleporter/README.md

```diff
@@ -7,8 +7,9 @@
   - [Fees](#fees)
     - [Message Receipts and Fee Redemption](#message-receipts-and-fee-redemption)
   - [Required Interface](#required-interface)
-  - [Teleporter Contract Deployment](#teleporter-contract-deployment)
   - [Message Delivery and Execution](#message-delivery-and-execution)
+  - [Resending a Message](#resending-a-message)
```

Comment:
Nit: Github provides an outline/table of contents by default. Can we remove the manual one while we're here so it doesn't fall out of date in the future?

![image](https://github.com/user-attachments/assets/2b921244-db03-438b-973e-99731d4f9a2d)

Reply:
100%